tests/jepsen/src/jepsen/memgraph/nemesis.clj

```diff
@@ -26,17 +26,28 @@
   [f]
   {:type :info :f f})
 
-(defn full-generator
-  "Construct nemesis generator."
+
+(defn nemesis-events
+  "Constructs events for nemesis based on provided options."
   [opts]
-  (->> [(when (:kill-node? opts)
+  (apply concat
+      [(when (:kill-node? opts)
           [(cycle (map op [:kill-node :restart-node]))])
-        (when (:partition-halves? opts)
+      (when (:partition-halves? opts)
           [(cycle (map op [:start-partition-halves :stop-partition-halves]))])]
-       (apply concat)
-       gen/mix
-       (gen/stagger (:interval opts))
-       (gen/phases (gen/sleep 60))))
+  ))
+
+
+(defn full-generator
+  "Construct nemesis generator."
+  [opts]
+  (gen/phases
+   (gen/log "Waiting replicas to get data from main.")
+   (gen/sleep 40)
```

Comment:
Willl we wait 40 seconds on each phase or just before the 1st phase?

Reply:
it should be just before the 1st phase